src/edu/ucsb/nceas/metacat/doi/datacite/EML2DataCiteFactory.java

```diff
@@ -0,0 +1,258 @@
+            can = true;
+        }
+        logMetacat.debug("EML2DataCitFactory.canProcess - If this factory can process the xml with the name space "+namespace+"? "+can);
+        return can;
+    }
+    
+    /**
+     * Method to generate the data cite xml document
+     */
+    @Override
+    public String generateMetadata(Identifier identifier, SystemMetadata sysmeta) throws InvalidRequest, ServiceFailure {
+        if(identifier != null && sysmeta != null) {
+            try {
+                
+                DataPackage emlPackage = getEMLPackage(sysmeta);
+                if (emlPackage != null) {
+                    String language = emlPackage.getLanguage();
+                    Document doc = generateROOTDoc();
+                    
+                    //identifier
+                    String scheme = DOI;
+                    String id = removeIdSchemePrefix(identifier.getValue(), scheme);
+                    addIdentifier(doc, id, scheme);
+                    
+                    //creator
+                    appendCreators(sysmeta.getRightsHolder(), emlPackage, doc);
+                    
+                    //title
+                    String title = emlPackage.getTitle();
+                    if(title == null || title.trim().equals("")) {
+                        throw new InvalidRequest(INVALIDCODE, "The datacite instance must have a title. It can't be null or blank");
+                    }
+                    appendTitle(title, doc, language);
+                    
+                    //publish
+                    Node node = MNodeService.getInstance(null).getCapabilities();
+                    addPublisher(doc,node.getName());
+
+                    //publication year
+                    SimpleDateFormat sdf = new SimpleDateFormat("yyyy");
+                    String year = sdf.format(sysmeta.getDateUploaded());
+                    addPublicationYear(doc, year);
+                    
+                    //subjects (keywords)
+                    List<String> subjects = emlPackage.getKeywords();
+                    if(subjects != null) {
+                        for(String subject : subjects) {
+                            appendSubject(subject, doc, language);
+                        }
+                    }
+                    
+                    //language
+                    addLanguage(doc, language);
+                    
+                    
+                    //resource type
+                    //String resourceType = lookupResourceType(sysmeta);
+                    String resourceType = null; //only set the attribute to "dataset"
+                    addResourceType(doc, DataCiteProfileResourceTypeValues.DATASET.toString(), resourceType);
+                    
+                    //version
+                    
+                    //description (abstract)
+                    String description = emlPackage.getAbsctrac();
```

Comment:
Refactor `getAbstrac()` to `getAbstract()`.  If this is out of scope because the method is defined elsewhere, let's at least create a ticket to fix it later.